applications/RANSModellingApplication/rans_modelling_application_variables.cpp

```diff
@@ -0,0 +1,38 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya (https://github.com/sunethwarna)
+//
+
+#include "rans_modelling_application_variables.h"
+
+namespace Kratos
+{
+KRATOS_CREATE_VARIABLE( double, TURBULENT_KINETIC_ENERGY )
+KRATOS_CREATE_VARIABLE( double, TURBULENT_ENERGY_DISSIPATION_RATE )
+KRATOS_CREATE_VARIABLE( double, TURBULENT_KINETIC_ENERGY_RATE )
+KRATOS_CREATE_VARIABLE( double, TURBULENT_ENERGY_DISSIPATION_RATE_2 )
+KRATOS_CREATE_VARIABLE( bool, IS_CO_SOLVING_PROCESS_ACTIVE )
+KRATOS_CREATE_VARIABLE( double, RANS_Y_PLUS )
+KRATOS_CREATE_VARIABLE( double, RANS_AUXILIARY_VARIABLE_1 )
+KRATOS_CREATE_VARIABLE( double, RANS_AUXILIARY_VARIABLE_2 )
+KRATOS_CREATE_VARIABLE( double, WALL_SMOOTHNESS_BETA )
+KRATOS_CREATE_VARIABLE( double, WALL_VON_KARMAN )
+KRATOS_CREATE_VARIABLE( double, TURBULENCE_RANS_C_MU )
+KRATOS_CREATE_VARIABLE( double, TURBULENCE_RANS_C1 )
+KRATOS_CREATE_VARIABLE( double, TURBULENCE_RANS_C2 )
+KRATOS_CREATE_VARIABLE( double, TURBULENT_VISCOSITY_MIN )
+KRATOS_CREATE_VARIABLE( double, TURBULENT_VISCOSITY_MAX )
+KRATOS_CREATE_VARIABLE( double, TURBULENT_KINETIC_ENERGY_SIGMA )
+KRATOS_CREATE_VARIABLE( double, TURBULENT_ENERGY_DISSIPATION_RATE_SIGMA )
+
+KRATOS_CREATE_VARIABLE(Element::WeakPointer, PARENT_ELEMENT)
```

Comment:
doesn't this already exist in the core? I am quite sure

Reply:
yeah, I resorted to the existing variables. :)